sonic_data_client/db_client.go

```diff
@@ -1073,6 +1092,7 @@ func dbSingleTableKeySubscribe(c *DbClient, rsd redisSubData, updateChannel chan
 					}
 					key := subscr.Channel[prefixLen:]
 					newMsi[key] = fp
+					newMsi["delete"] = "null_value"
```

Comment:
TODO on above line1082 could be removed.

Reply:
Done